### DISCUSSION TITLE
Participant leave notification.

### DISCUSSION THREAD

**Discussion by ThilinaV98** [mynaparrot/plugNmeet-server#338] [Q&A] [answered]
1. If a participant in the meeting closes his browser and leaves the meeting, it will show that he is in the meeting, but he is not actually in the meeting.
2. If a participant who has the camera turned on in the meeting closes his browser and leaves the meeting, the camera freezes and shows that he is in the meeting, but he is not actually in the meeting.
3. If a participant leaves the meeting, a notification will not be displayed.

**Comment by jibon57** [comment, marked as answer]
Clearing status of users may take few seconds. This is expected behavior.
